src/Interface/Modules/Render/ViewSceneControls.ui

```diff
@@ -108,6 +108,46 @@
         </item>
        </layout>
       </widget>
+      <widget class="QWidget" name="CameraLocks">
+       <attribute name="title">
+        <string>Camera Locks</string>
+       </attribute>
+       <layout class="QGridLayout" name="gridLayout_9">
+        <item row="7" column="0">
+         <widget class="QTreeWidget" name="viewSceneTreeWidget_">
+          <property name="columnCount">
+           <number>1</number>
+          </property>
+          <column>
+           <property name="text">
+            <string>Groups</string>
+           </property>
+          </column>
+         </widget>
+        </item>
+        <item row="1" column="0">
+         <layout class="QHBoxLayout" name="horizontalLayout_3">
+          <item>
+           <widget class="QPushButton" name="addGroup_">
+            <property name="text">
+             <string>Add Group</string>
+            </property>
+           </widget>
+          </item>
+          <item>
+           <widget class="QPushButton" name="removeGroup_">
+            <property name="text">
+             <string>Remove Group</string>
```

Comment:
```suggestion
             <string>Remove First Group</string>
```

Reply:
since the first group is always deleted, this will help reduce confusion.  we can make an issue to correct this to how we want it.